src/baseline_bots/bots/dipnet/no_press_bot.py

```diff
@@ -25,3 +25,7 @@ def gen_orders(self):
         orders = yield self.brain.get_orders(self.game, self.power_name)
         self.orders.add_orders(orders, overwrite=True)
         return self.orders.get_list_of_orders()
+
+    @gen.coroutine
```

Comment:
What exactly is the need for adding `@gen.coroutine`? Since this seems to get applied to various bots, would it make sense to have a wrapper class that does this for any bot?

Reply:
Can you try `async def` instead?

Reply:
I also don't have a strong understanding around tornado. I had tried to get my hands dirty by trying out different things - asnyc def, gen.coroutine and gen.coroutine has been working fine. So, I just didn't want to meddle with that even if it is an older style. Is it possible that you could assign someone else to try this out in a separate PR?

Reply:
Yep